src/utils/apiDataMapping.ts

```diff
@@ -0,0 +1,173 @@
+export const getLanguageEmoji = (lang: string): string => {
+  switch (lang) {
+    case 'angielski':
+      return '🇬🇧';
+    case 'niemiecki':
+      return '🇩🇪';
+    case 'francuski':
+      return '🇫🇷';
+    case 'rosyjski':
+      return '🇷🇺';
+    default:
+      return lang;
+  }
+};
+
+export const getSchoolTypeFromRspoInstitutionTypeId = (id: number) => {
```

Comment:
```js
const mapping = {
  90: 'Bednarska Szkoła Realna',
  58: 'Biblioteki pedagogiczne',
  …
}

return mapping[id];
```

Reply:
If you want the default behaviour you can catch key error and return the default then

Reply:
Done